src/rpc/server/scala/serv/src/omega-edit-grpc-server.js

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
Add `#!/usr/bin/env node` here.  It's a standalone script.  Might as well make it a little easier to run for non-Windows.